src/main/java/com/minecolonies/core/colony/buildings/modules/MinimumStockModule.java

```diff
@@ -57,8 +57,9 @@ public class MinimumStockModule extends AbstractBuildingModule implements IMinim
     private int minimumStockSize()
     {
         final double increase = 1 + building.getColony().getResearchManager().getResearchEffects().getEffectStrength(MINIMUM_STOCK);
+        final int buildingLevelOr5 = building.getMaxBuildingLevel() != 0 ? building.getBuildingLevel() : 5;
 
-        return (int) (building.getBuildingLevel() * STOCK_PER_LEVEL * increase);
+        return (int) (buildingLevelOr5 * STOCK_PER_LEVEL * increase);
```

Comment:
Again, please remove this change and override `getBuildingLevel()` of the postbox instead